src/visualization/types/Map/view.tsx

```diff
@@ -120,11 +120,7 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
         type: 'pointMap',
         colorDimension: {label: 'Value'},
         colorField: '_value',
-        colors: [
-          {type: 'min', hex: InfluxColors.Star, id: '0'},
-          {value: 50, hex: InfluxColors.Star, id: '1'},
-          {type: 'max', hex: InfluxColors.Star, id: '2'},
-        ],
+        colors: DEFAULT_THRESHOLDS_GEO_COLORS,
```

Comment:
I think you need to update line 130 to use the `DEFAULT_THRESHOLD_GEO_COLORS`.

Reply:
did it!